newrelic_python_agent/plugins/base.py

```diff
@@ -275,7 +275,7 @@ def metric_payload(self, value, min_value=None, max_value=None, count=None,
         if not value:
             value = 0
 
-        if isinstance(value, basestring):
+        if isinstance(value, ("".__class__, u"".__class__)):
```

Comment:
Wouldn't it be better to use `six` for this? I find this method a tad hard to read to be honest :P

`if isinstance(value, six.string_types)`

Reply:
Personally I think `six` is available nearly everywhere because of other dependencies so it shouldn't be that much of a problem, but lets ask @Rockstar04 as well :)
Alternatively we could simply integrate `six` into the library since it's a single file anyhow: https://github.com/benjaminp/six/blob/master/six.py

Reply:
I'm pretty sure I've mentioned before, I'm from from familiar with the Python ecosystem. I do know enough that `python-six` is available  all the way back to Ubuntu 12.04, and a quick search looks like good coverage in RPM based systems as well. I'd be fine with adding it as a dependency.

Reply:
OK, I updated it to use six and included six>=1.5 in setup.py. wasn't sure what version, but awscli uses six>=1.5 so that seemed good to me.